src/main/java/pl/javastart/task/Thing.java

```diff
@@ -0,0 +1,11 @@
+package pl.javastart.task;
+
+public class Thing {
+    String name;
+    String describe;
+
+    public Thing(String name, String describe) {
+        this.name = name;
+        this.describe = describe;
+    }
+}
```

Comment:
Domyślam się, że chodziło o to, żeby wyciągnąć część wspólną i nie powtarzać `name` i `decribe` w innych klasach. Takie podejście nie jest jednak wskazane. Przenieś proszę te pola bezpośrednio do klas gdzie są potrzebne.